index.html

```diff
@@ -0,0 +1,165 @@
+          taking place in October, in Korea.
+        </p>
+        <p class="time">2022.10.15(thu) ~ 16(fri)</p>
+        <p class="address">
+          @ National Museum of Korea, Art Center Nabi and more
+        </p>
+      </section>
+
+      <!--Main Program Section-->
+      <section id="main-program">
+        <p class="title">Main Program</p>
+        <hr />
+        <div class="main-program">
+          <div class="lecture">
+            <img src="./assets/logos/p5.png" alt="lecture logo" />
+            <h3>Lecture</h3>
+            <p>
+              listen to the speakers from various countries about the messeges
+              of sharing and opening.
+            </p>
+          </div>
+          <div class="ccexhibition">
+            <img src="./assets/logos/p1.png" alt="cc Exhibition logo" />
+            <h3>CC Exhibition</h3>
+            <p>
+              Appreciate various creations applying CC license of artists,
+              organized from Art Center Nabi.
+            </p>
+          </div>
+          <div class="forum">
+            <img src="./assets/logos/p2.png" alt="Forum logo" />
+            <h3>Forum</h3>
+            <p>
+              Have the time to share your thoughts and opinions with experts for
+              each topic.
+            </p>
+          </div>
+          <div class="workshop">
+            <img src="./assets/logos/p3.png" alt="workshop logo" />
+            <h3>Workshop</h3>
+            <p>
+              try creating your own work using open source license rather than
+              just watching at it.
+            </p>
+          </div>
+          <div class="ccignite">
+            <img src="./assets/logos/p4.png" alt="cc ignite logo" />
+            <h3>CC Ignite</h3>
+            <p>
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+        <p class="desk">See the whole program</p>
+        <p class="mob">Join CC Global Summit 2022</p>
+      </section>
+      <!--Featured Speakers Section-->
+      <section id="featured-speaker">
+        <div id="speakers"></div>
+      </section>
+      <!--Partners Section-->
+      <section id="partner">
+        <p class="title">Partners</p>
+        <hr />
+        <div class="partner">
+          <div class="partners">
+            <img src="./assets/logos/amazon.png" alt="amazon logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/deezer.png" alt="deezer logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/linkedin.png" alt="linkedin logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/paypal.png" alt="paypal logo" />
+          </div>
+          <div class="partners">
+            <img src="./assets/logos/airbnb.png" alt="airbnb logo" />
+          </div>
```

Comment:
Please ensure that the pages look almost identical to the original design. This section should contain a line of text, and two images with text centered over them.


Your Implementation | Figma Design
:---:|:---:
![your_design](https://user-images.githubusercontent.com/10332499/153335915-9f4d5077-b16b-4694-ab9f-691c311f706c.PNG)
 | ![design](https://user-images.githubusercontent.com/10332499/153335941-fce4522f-e2f7-48c7-a38e-a4d65d9971c1.PNG)

Reply:
yes, you right
I did solve  the issue